dash/development/component_generator.py

```diff
@@ -82,13 +89,17 @@ def generate_components(components_source, project_shortname,
     )
 
     with open(os.path.join(project_shortname, 'metadata.json'), 'w') as f:
-        json.dump(metadata, f)
+        json.dump(metadata, f, sort_keys=True)
```

Comment:
:rewind: Sorting the key reorder the keys, the ordered dict will keep the right order alone.